scripts/performance/common.py

```diff
@@ -157,6 +158,37 @@ def push_dir(path: str = None) -> None:
     else:
         yield
 
+def retry_on_exception(function, retry_count=3, retry_delay=5, retry_delay_multiplier=1, retry_on_exception=None):
```

Comment:
Would it make sense to have `retry_on_exception=Exception` instead code on L178-179?

Reply:
Updated! I agree that this seems like a better approach.